src/components/application_manager/include/application_manager/resumption/resume_ctrl.h

```diff
@@ -115,11 +115,34 @@ class ResumeCtrl {
   virtual void OnSuspend() = 0;
 
   /**
+   * @brief Processes resumption data after receiving signal "Ignition Off"
+   */
+  virtual void OnIgnitionOff() = 0;
+
+  /**
    * @brief Processes resumption data after receiving signal "Awake"
    */
   virtual void OnAwake() = 0;
 
   /**
+   * @brief Retrieves value of is_suspended_
```

Comment:
@dev-gh  done in https://github.com/smartdevicelink/sdl_core/pull/1604/commits/1f59584e622b589eb2b4e17926b0cbb10463bcd9